src/main/java/com/researchspace/dataverse/api/v1/UsersOperations.java

```diff
@@ -29,19 +29,18 @@ public interface UsersOperations {
 
     /**
      * Get token expiration date.
-     * @return java.util.Date token expiration date.
+     * @return java.time.LocalDateTime token expiration.
      * @throws ParseException
      * @throws RestClientException
      */
-    Date getTokenExpirationDate() throws ParseException;
+    LocalDateTime getTokenExpirationDate() throws ParseException;
 
     /**
-     * Recreate user token.
-     * @return new token
-     * @throws Exception
+     * Get token expiration text returned by dataverse.
+     * @return String as the form of <Token XXXXX expires on yyyy-MM-dd HH:mm:ss.SSS>.
+     * @throws ParseException
+     * @throws RestClientException
      */
-    String recreateToken();
```

Comment:
deleting /renaming this will break existing clients. Solution is to keep the original method signature and add your method as a new method - it looks like it is doing something different?

Reply:
The method was finally not developed. We considered, that in our use, it was better to create the token ourselves when it would expire.